### PR TITLE
Feature/price config

### PR DESCRIPTION
**Pull request by nicmr**
Removed:
 - serve subcommand (#44 )

Added:
 - support for margin flag to allow price configuration

### CODE REVIEW

File: cmd/invoice.go [hunk 1]
```diff
@@ -45,6 +49,11 @@ func init() {
 		log.Fatal("Unable to bind viper to flag:", err)
 	}
 
+	invoiceCmd.Flags().Float64Var(&margin, "margin", 1.00, "The margin that should be added on top of resource costs as ops compensation")
```
Comment (markushinz) [inline cmd/invoice.go]: By the description I would have expected a margin of 0.0 to be "free of charge" and a margin of 1.0 to be "double the actual costs". Please change the description - or even better the implementation (see other comment)

File: cmd/invoice.go [hunk 2]
```diff
@@ -65,7 +74,23 @@ func cost() {
...
+	apiResult, err := costapi(parsedMonth)
+
+	// Apply the margin
+	for i, entry := range apiResult.CsvEntries {
+		margin := viper.GetFloat64("margin")
+		amount, err := strconv.ParseFloat(entry.Amount, 64)
+		if err != nil {
+			log.Fatal("unable to parse cost value returned by AWS: ", err)
+		}
+		total := amount * margin
```
Comment (markushinz) [inline cmd/invoice.go]: `total := amount * (1.00 + margin)`

File: cmd/invoice.go (hunk 1 above)
Comment (nicmr) [reply cmd/invoice.go]: implemented

File: cmd/invoice.go (hunk 2 above)
Comment (nicmr) [reply cmd/invoice.go]: implemented

**Pull request merged**